tests/rptest/test_suite_quick.yml

```diff
@@ -13,3 +13,4 @@ quick:
 
   excluded:
   - tests/librdkafka_test.py
+  - test/openmessaging_benchmark_test.py
```

Comment:
is this temporary?

Reply:
No.
As I understand, we ran full test_suite in CI
But local run of this benchmark ully uses resources resource of computer, so it just freezes

Reply:
is there a way to reduce the size so that it can run locally? we have a sort of flag in ducktape that is set depending on the execution environment (e.g. local vs CI). if this doesn't really work then excluding it is ok.

Reply:
There is a way. I can set produce rate to some constant for local environment